screens/Reader.js

```diff
@@ -724,29 +703,25 @@ class Reader extends React.Component {
                 />
               )}
               <Slider
-                style={[
-                  { flex: 1, marginLeft: 25, marginRight: 25, marginTop: 10 },
-                ]}
+                style={[{ flex: 1, marginLeft: 25, marginRight: 25, marginTop: 10 }]}
                 minimumTrackTintColor="#BFBFBF"
                 maximumTrackTintColor="#464646"
                 thumbTintColor="#fff"
```

Comment:
I know this is not the change in this PR veerji, but still if you are refactoring other color strings please consider changing this one as well.

Reply:
fixed